Simplified/Book/UI/NYPLBookCellDelegate.m

```diff
@@ -30,7 +30,7 @@ @interface NYPLBookCellDelegate () <RefreshDelegate>
 
 @property (nonatomic) NSTimer *timer;
 @property (nonatomic) NYPLBook *book;
-@property (nonatomic) id<AudiobookManager> manager;
+@property DefaultAudiobookManager *manager;
```

Comment:
There's a huge change that I'm missing something here but it looks like we're breaking the dependency inversion principle.

Reply:
My changes are pretty minimal, so I'm guessing you mean with the existing implementation?

Reply:
oops sorrry I meant CHANCE not change @ettore

Reply:
hey Raman can you explain what you mean? this is for a private ("private" in objc terms that is) property so calling things with their actual name is not making things more tightly dependent than using a protocol.

Reply:
you're right. I missed the 'private' part.